<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conf_Multi</class>
 <widget class="QMainWindow" name="Conf_Multi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>848</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuraci칩n Multiclasificador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="reset_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>670</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Reglas">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>421</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Reglas</string>
    </property>
    <widget class="QComboBox" name="Regla">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>IGUAL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DISTINTO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MAYOR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MENOR</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Etiqueta Referencia</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Etiq_Ref">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>60</width>
       <height>32</height>
      </rect>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Peso</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Peso">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>62</width>
       <height>32</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="Anadir_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>A침adir</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Clasificador</string>
    </property>
    <widget class="QRadioButton" name="Distancias_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Distancias</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="KNN_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>KNN</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Histograma_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Histograma</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Boosting_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Boosting</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="GAUSS_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Bayesiano (GAUSS)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="SVM_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>SVM</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="DTrees_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>DTrees</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Neuronal_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Neuronal</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="EM_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Expectation Maximization</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RTrees_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>RTrees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Anadir">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>250</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>A침adir</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="Informacion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>831</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Tipo</string>
    </property>
    <widget class="QRadioButton" name="Cascada_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Cascada</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="Votacion_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>115</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Votaci칩n</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Aceptar_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>670</y>
      <width>96</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
